queries.js

```diff
@@ -24,7 +24,9 @@ module.exports = (self, query) => {
           }
         },
         launder(value) {
-          return self.apos.launder.booleanOrNull(value);
+          return self.apos.launder.booleanOrNull(
+            value === '' ? null : value
```

Comment:
Should be OK. I still don't completely understand why `any` can't be passed but I think this has good enough backwards compatibility and I'm not going to keep dragging you back 😆

Reply:
`any` works  well but I find it has more sense to pass `null` explicitly, because passing `any` needs us to take a look at the `booleanOrNull` implementation to understand that it will result in `null`.
I also chose `null` because in the `booleanOrNull` implementation, `null` is handled at the very beginning of the function whereas  `any` is handled after loads of conditions.. that's micro-optimization but this, added to my first point, I  chose `null`